packages/x-data-grid/src/components/GridSkeletonLoadingOverlay.tsx

```diff
@@ -0,0 +1,135 @@
+import * as React from 'react';
+import { styled } from '@mui/system';
+import {
```

Comment:
```suggestion
import useForkRef from '@mui/utils/useForkRef'
import composeClasses from '@mui/utils/composeClasses'
```

We should probably update the lint config to enforce it.

Reply:
Thanks, fixed here https://github.com/mui/mui-x/pull/13293/commits/e98680ebdff3c49683772c7cfdae2d07df52fe90